src/libpriv/rpmostree-core.c

```diff
@@ -1656,10 +1658,24 @@ relabel_rootfs (OstreeRepo        *repo,
                 GCancellable      *cancellable,
                 GError           **error)
 {
+  guint n_relabeled = 0;
+  gboolean ret;
+  
   /* NB: this does mean that / itself will not be labeled properly, but that
    * doesn't matter since it will always exist during overlay */
-  return relabel_dir_recurse_at (repo, dfd, ".", "/", sepolicy,
-                                 out_changed, cancellable, error);
+  ret = relabel_dir_recurse_at (repo, dfd, ".", "/", sepolicy,
+                                &n_relabeled, cancellable, error);
+  /* Print how many things changed to help debug things in the future
+   * and guide how important it is to accurately cache things.
+   */
+  if (ret)
+    {
+      *out_changed = n_relabeled > 0;
+      if (*out_changed)
+        g_print ("%u files relabeled\n", n_relabeled);
```

Comment:
This doesn't get printed properly because the console is under lock during that time.

Also, though I do agree that it can be useful information, I feel like it would be too much details most of the time. Maybe a debug env var instead?